client/examples/http_client.rs

```diff
@@ -158,6 +158,41 @@ async fn ping(
     response::ok_json::<Pong>(&msg)
 }
 
+#[get("/disconnect/{node_id}")]
+async fn disconnect(
+    node_id: web::Path<NodeId>,
+    client_sender: web::Data<ClientWrap>,
+    messages: web::Data<Messages>,
+) -> actix_web::Result<HttpResponse> {
+    log::trace!("[disconnect]: Disconnecting {}", node_id);
+    let node_id = node_id.into_inner();
+
+    let msg = client_sender
+        .run_async(move |client: Client| async move {
+            let r = messages.request();
+            let msg = format!("Close Session:{}", r.id());
+            log::debug!("Sending close session message: {}", msg);
+            let _s = client.sessions().await;
```

Comment:
It is done to initialize internal sessions collection or sth?

Reply:
It's not needed -> I'll remove it